dotnet-local.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ -d "bin/Release/dotnet" ]; then
```

Comment:
I'd use `-x "bin/Release/dotnet/dotnet"` here, so that we know that the binary both exists and is executable